danielMunteanu/src/main/java/Order.java

```diff
@@ -0,0 +1,67 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+class Order {
+    public long id;
+    int counter;
+    public String date;
+    public User user;
+    List<Item> items;
+
+    public Order(User user, List<Item> items) {
+        // Set ID
+        this.id = ++counter;
+
+        // Set user
+        this.user = user;
+
+        // Set Date
+        LocalDateTime now = LocalDateTime.now();
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSS");
+        this.date = now.format(formatter);
+
+    }
+
+    public Order() {
+    }
+
+    public void setId(int id) {
```

Comment:
do we need SetId method meanwhile we have already a defined id within the constructor?

Reply:
removed